### ISSUE TITLE
allow to use it as Hands-free system

### ISSUE THREAD

**Issue by Mannshoch** [xbmc/Kore#669]
## Feature suggestion
**Detailed Description**

It would be nice if im able to cumunicat from a Mobile phone to a TV user.
May also as singel channel or as walkie-talkie only transfer if you longpress a button.

### Context

* I would use it to shout my children because they have to finish tv and appear at table for dinner. 
* Use it as loudspeaker at parties. 

### Possible Implementation

Kind of a walkie-talkie, longpress the middle button on the steering cross.
* Initiated on Kodi-app, have a  mic-icon on the sidebar. Tick on it a hovering button appear which either show available devices or if longpress open a channel to this user.
If phone is on silent mode you could not get contacted.

* Initiated from Tv show a list of available people. Then every longpress contact the same. Normal short click then also list other contacts. 
After 10min of not using the long click at first shows available devices. And last used is marked. If you still press the buton another 3s the last used contact get activated and the channel open.

**Comment by axelsimon** [comment]
One way this can / has been implemented is in [Kodimote](https://github.com/mzanetti/kodimote) where you can chose to share a multitude of things to Kodi from the app over UPnP, including live microphone input from the phone.

However, this does no mixing, so it just replaces whatever is playing on Kodi with this new stream.

**Comment by Mannshoch** [comment]
> However, this does no mixing, so it just replaces whatever is playing on Kodi with this new stream.
that's what I hope for.

It seems that this Repository is death since 2015.
I would Propose to integrate functionality this into your app.

**Comment by axelsimon** [comment]
Yes, it's just an example. It's an app for a different OS too (Sailfish vs. Android) so again, just used as an example.

I'm not sure there is much point in just making it possible to send mic input over UPnP, it's not a great solution. This should be thought out more.

**Comment by Mannshoch** [comment]
Since I created this request a new interesting idea appear. I have a geriatric family member with no flair for Computer. I'm looking at moment for a solution to let him do a voice call with the family in an easy way. I look for a Messenger I could implement into kodi so this person may use the TV for Voice, maybe Videocalls he may use with his TV remote. I have a nextcloud-talk server but other messaging apps like riot, conversation may cloud also be a tool for this.
